devlib/target.py

```diff
@@ -945,10 +945,7 @@ def capture_screen(self, filepath):
 
     def _resolve_paths(self):
         if self.working_directory is None:
-            if self.connected_as_root:
-                self.working_directory = '/root/devlib-target'
-            else:
-                self.working_directory = '/home/{}/devlib-target'.format(self.user)
+            self.working_directory = self.execute("pwd".format(self.user)).strip()
```

Comment:
`.format(self.user)` isn't required here.

Reply:
oops. leftover from previous version. sorry about that.